source/ios/tools/AdaptiveCards.podspec

```diff
@@ -1,20 +1,25 @@
-  
-Pod::Spec.new do |spec|
+od::Spec.new do |spec|
```

Comment:
**P**od here, missing **P**

Reply:
thanks

Reply:
it's weird. i noticed it happened again with another commit; it could be due to tmux i'm usig.